src/live.asp.net/Services/YouTubeShowsService.cs

```diff
@@ -22,6 +25,8 @@ public class YouTubeShowsService : IShowsService
     {
         public const string CacheKey = nameof(YouTubeShowsService);
 
+        public int ResultsCount { get; } = 3 * 8;
```

Comment:
now `_resultsCount` is a private const, but I would like to know what @DamianEdwards thinks about my proposal of *injecting* the value into the `IShowsService` implementation.